openstef/model/regressors/linear_quantile.py

```diff
@@ -89,6 +92,9 @@ def __init__(
                 "Cannot train quantile model as 0.5 is not in requested quantiles!"
             )
 
+        if clipped_features is None:
+            clipped_features = ["APX"]
```

Comment:
We need to be wary of https://github.com/OpenSTEF/openstef/pull/578 which changes "APX" everywhere, so we woud need to do this here as well

Reply:
Good to know. Once this is merged I will push an addition to that PR.